### ISSUE TITLE
Feature Request: Add area to add a description on the series Page.

### ISSUE THREAD

**Issue by kimwhite** [strangerstudios/pmpro-series#87]
Series Page will show Post Name and Availablity but would benefit from a way to add a description to this list.

Example
<img width="747" alt="Screen Shot 2021-06-21 at 4 05 15 PM" src="https://user-images.githubusercontent.com/8083630/122821957-78a12780-d2ab-11eb-910d-f47cc4bde87a.png">

**Comment by kimcoleman** [comment]
Would you think this could be the WordPress-generate post "excerpt" (if we show them to non-members? I don't think we'd add a new field / way to set a short description per item. But pulling in the core WP excerpt sounds interesting and we could use a filter to do so.

**Comment by cremigerhonig** [comment]
yes, also let me choose if they can see the upcoming videos full list right away or one by one

also: what happes if someone upgrade their level, the counting starts from 0?
